test/screenshot/infra/lib/cbt-api.js

```diff
@@ -527,6 +545,45 @@ https://crossbrowsertesting.com/account
     console.error(`${colorRetrying} (attempt ${colorAttemptCount} of ${colorMaxRetries})...`);
     console.error('');
   }
+
+  /**
+   * @param {!Array<!CbtSeleniumListItem|!CbtSeleniumInfoResponse>} sessions
+   * @param {string} description
+   * @private
+   */
+  logSeleniumSessions_(sessions, description) {
+    const jsonStr = JSON.stringify(sessions.map((session) => {
+      return {
+        selenium_test_id: session.selenium_test_id,
+        selenium_session_id: session.selenium_session_id,
+        start_date: session.start_date,
+        startup_finish_date: session.startup_finish_date,
+        finish_date: session.finish_date,
+        active: session.active,
+        state: session.state,
+        show_result_web_url: session.show_result_web_url,
+        show_result_public_url: session.show_result_public_url,
+        caps: session.caps,
```

Comment:
What is this? Judging by the externs change it was just added? I don't see it referenced anywhere else in this PR.

Reply:
It's always been present in the HTTP response, but I didn't add it to the externs file before for some reason.

I found it helpful to have this in the console when I was trying to match up Session IDs with Test IDs in the CBT UI.